packages/azure-kusto-data/src/models.ts

```diff
@@ -13,7 +13,7 @@ export enum WellKnownDataSet {
 type DateTimeParser = (value: string) => any;
 type TimeSpanParser = (value: string) => any;
 
-const defaultDatetimeParser: DateTimeParser = (t: string) => new Date(t);
+const defaultDatetimeParser: DateTimeParser = (t: string) => (t ? new Date(t) : null);
```

Comment:
So this method also should accept (t: string | null) ?

Reply:
Thanks!